### DISCUSSION TITLE
Use or lose "White border"?

### DISCUSSION THREAD

**Discussion by Jay-o-Way** [U-C-S/Hurl#161] [Polls]
There's a setting to hide it, but why is it still alive in the first place? My preference is to move more and more towards MS WinUI etc.

**Comment by U-C-S** [comment]
This is back when `v0.5` was first released, when I felt like pure acrylic is bit hard to notice. I doubt anyone would notice any difference without it 

So I would say having "White Border" disabled by default instead, atleast till we move to WinUI would be good enough
